pkg/apis/minio.min.io/v2/types.go

```diff
@@ -327,6 +327,13 @@ type TenantSpec struct {
 	//
 	// * `CONSOLE_SECRET_KEY` - The "Password" for the MinIO user +
 	//
+	// * You can set a built-in policy for `[*]_ACCESS_KEY/[*]_SECRET_KEY/[*]_POLICY`
```

Comment:
We should add samples in helm charts and kustomize example YAMLs?
For example, we can add entry in `helm/tenant/templates/tenant-configuration.yaml` for the additional secrets.